app/controllers/api/v3/games_controller.rb

```diff
@@ -7,18 +7,18 @@ def index
       return
     end
     @games = Game.search(params[:search]).includes(:categories)
-    render json: @games, each_serializer: Api::V3::GameSerializer, root: :games
+    render json: GameBlueprint.render(@games, view: :api_v3, root: :games)
   end
 
   def show
-    render json: @game, serializer: Api::V3::GameSerializer, root: :game
+    render json: GameBlueprint.render(@game, view: :api_v3, root: :game, toplevel: :game)
   end
 
   private
 
   def set_game
-    @game = Game.joins(:srdc).includes(:categories).find_by(speedrun_dot_com_games: {shortname: params[:id]}) || Game.includes(:categories).find(params[:id])
```

Comment:
Any specific reason for removing these `includes` calls? Does Blueprinter handle it for us now?